.github/workflows/build.yaml

```diff
@@ -13,25 +16,32 @@ jobs:
     - name: install ninja
       run: sudo apt install ninja-build
     - uses: actions/checkout@v2
+    - name: clone-vcpkg
+      run: >
+        git clone --quiet --shallow-since 2021-02-01 \
```

Comment:
If the goal is to get a specific snapshot of the vcpkg repo, you may be able to use extract a tarball of the exact hash you want, instead of needing to do a shallow clone w/ a hard-coded date, then sync to the hash you want. Maybe something like:

```sh
mkdir /tmp/vcpkg && curl -sSL "https://github.com/microsoft/vcpkg/archive/5214a247018b3bf2d793cea188ea2f2c150daddd.tar.gz" | tar -C /tmp/vcpkg --strip-components=1 -zxf -

```

I was thinking about doing this in `google-cloud-cpp`, so if this is not a good idea, do let me know :-)

Reply:
That sounds better, will do.